client-v2/src/shared/components/article/ArticleBody.tsx

```diff
@@ -241,10 +244,9 @@ const articleBodyDefault = React.memo(
           ) : (
             <DraggableArticleImageContainer id={uuid} canDrag={canDragImage}>
               <ThumbnailSmall
-                style={{
-                  backgroundImage: `url('${thumbnail}')`,
-                  opacity: imageHide ? 0.5 : 1
-                }}
+                imageHide={imageHide}
+                url={thumbnail}
+                isDraggingImageOver={isDraggingImageOver}
```

Comment:
Extracting the other two variables here (thumbnail and imageHide) so we can get rid of this inline style

Reply:
👍